packages/babel-helper-create-class-features-plugin/src/fields.js

```diff
@@ -224,11 +224,18 @@ const privateNameHandlerSpec = {
     }
 
     if (isMethod) {
-      if (isAccessor) {
+      if (getId) {
         return t.callExpression(file.addHelper("classPrivateFieldGet"), [
           this.receiver(member),
           t.cloneNode(id),
         ]);
+      } else if (setId) {
+        if (file.availableHelper("writeOnlyError")) {
+          return t.callExpression(file.addHelper("writeOnlyError"), [
+            t.stringLiteral(name),
+          ]);
+        }
+        return;
```

Comment:
Should this throw an error?

Reply:
We should fallback to what we did in `getId` when `writeOnlyError` is not available, but we could nudge users to update `@babel/helpers` via warning messages, as it deems a runtime error that most users should avoid to use.